lib/components/Input/text-field-with-steppers/index.tsx

```diff
@@ -7,55 +7,60 @@ import {
     LabelPairedMinusSmRegularIcon,
 } from "@deriv/quill-icons";
 import "./text-field-with-steppers.scss";
-import { TLeftOrRight } from "@types";
 
 export interface TextFieldWithSteppersProps extends ComponentProps<typeof Input> {
-    unit?: string;
     decimals?: number;
-    unitPlacement?: TLeftOrRight;
+    unitLeft?: string;
+    unitRight?: string;
 }
 
 export const TextFieldWithSteppers = forwardRef<HTMLInputElement, TextFieldWithSteppersProps>(
     (props, ref) => {
-        const { value, onChange, disabled, textAlignment, placeholder = 0, unit = '', decimals = 2, unitPlacement = 'left' } = props;
+        const { value, onChange, disabled, textAlignment, placeholder = 0, unitLeft = '', unitRight = '', decimals = 2, ...rest } = props;
 
         const getFormatValue = (value: number) => parseFloat(value.toFixed(decimals));
 
         const steppersSectionRight = (
             <>
-                {textAlignment !== 'center' && (<button className={clsx("quill-input-steppers-decrement", disabled && 'quill-input-steppers--disabled')}
+                {textAlignment !== 'center' && (<button className={clsx("quill-input-steppers-button", "quill-input-steppers-button--decrement")}
                     onClick={() => {
                         if (!disabled) {
                             onChange?.({
                                 target: { value: getFormatValue(Number(value) - 1) },
                             } as unknown as React.ChangeEvent<HTMLInputElement>);
                         }
-                    }}>
+                    }}
+                    disabled={disabled}
+                    >
                     <LabelPairedMinusSmRegularIcon />
                 </button>)}
-                <button className={clsx("quill-input-steppers-increment", disabled && 'quill-input-steppers--disabled')}
+                <button className={clsx("quill-input-steppers-button", "quill-input-steppers-button--increment")}
                     onClick={() => {
                         if (!disabled) {
                             onChange?.({
                                 target: { value: getFormatValue(Number(value) + 1) },
                             } as unknown as React.ChangeEvent<HTMLInputElement>);
                         }
-                    }}>
+                    }}
+                    disabled={disabled}
+                    >
                     <LabelPairedPlusSmRegularIcon />
                 </button>
             </>
         );
 
         const steppersSectionLeft = (
             <>
-                <button className={clsx("quill-input-steppers-decrement", disabled && 'quill-input-steppers--disabled')}
+                <button className={clsx("quill-input-steppers-button", "quill-input-steppers-button--decrement")}
```

Comment:
Maybe if `jsx` with `button`  is used in multiple places we can create small reusable component?
Like component, which will take as props `className`, `icon` and `- 1` or `+ 1` for `onClick` function calculations?

Reply:
It's totally not worth it, it's used 2 times in its existence